libraries/ESP8266WiFi/src/ESP8266WiFiSTA.cpp

```diff
@@ -309,6 +309,13 @@ bool ESP8266WiFiSTAClass::config(IPAddress local_ip, IPAddress arg1, IPAddress a
     return false;
   }
 
+#if LWIP_VERSION_MAJOR != 1 && !CORE_MOCK

```

Comment:
Please move this block down and merge with the one at the bottom. There doesn't seem to be a need to have a separate block up here.

Reply:
I can't. `wifi_get_ip_info()` would get `wifi_set_ip_info()`'s result (called just below).

Reply:
Ah yes you're right.